addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -134,6 +134,20 @@ export class ImageField extends Component {
 export const imageField = {
     component: ImageField,
     displayName: _lt("Image"),
+    supportedOptions: [
+        {
+            name: "size",
+            label: _lt("Size"),
+            type: "selection",
+            getChoices() {
```

Comment:
suggestion (not sure about that): like we did for relatedFields, fieldDependencies..., getChoices could simply be `choices` which can be an array (like for this usecase) or a function returning an array for dynamic cases like the one below